services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -294,10 +294,14 @@ async def remove_users_manually_marked_as_guests(
         )
 
 
+def _get_project_db_api(app: web.Application) -> ProjectDBAPI:
+    return app[APP_PROJECT_DBAPI]
```

Comment:
You do not need this, it is already "integrated" in ``ProjectDBAPI``

SEE https://github.com/ITISFoundation/osparc-simcore/blob/3f32033c1bceadb1e20d739a7aed510f748ddd55/services/web/server/src/simcore_service_webserver/projects/projects_db.py#L184


just 

```python
     db = ProjectDBAPI.get_from_app_context(app)
```